pkg/component/gardener/discoveryserver/discoveryserver_test.go

```diff
@@ -84,6 +91,46 @@ var _ = Describe("GardenerDiscoveryServer", func() {
 	BeforeEach(func() {
 		ctx = context.Background()
 
+		workloadIdentityPrivateKey = []byte(`-----BEGIN PRIVATE KEY-----
```

Comment:
Should we generate these instead of hardcoding them? If speed is of the matter we can use elliptic curve keys which are fast to generate or generate a single key before the suite.

Reply:
We typically overwrite the generate functions used by secrets manager to speed up unit tests: https://github.com/gardener/gardener/blob/aed268d17c6ed0e0b146e90d5bf0086de4958c5f/pkg/component/apiserver/apiserver_suite_test.go#L24-L28